onnxruntime/test/hosting/integration_tests/model_zoo_tests.py

```diff
@@ -0,0 +1,101 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import unittest
+import random
+import os
+import test_util
+import sys
+
+class ModelZooTests(unittest.TestCase):
+    server_ip = '127.0.0.1'
+    server_port = 54321
+    url_pattern = 'http://{0}:{1}/v1/models/{2}/versions/{3}:predict'
+    hosting_app_path = ''  # Required
+    log_level = 'verbose'
+    server_ready_in_seconds = 10
+    server_off_in_seconds = 100
+    need_data_preparation = False
+    need_data_cleanup = False
+    model_zoo_model_path = ''  # Required
+    model_zoo_test_data_path = ''  # Required
+    supported_opsets = ['opset_7', 'opset_8', 'opset_9']
+    skipped_models = ['tiny_yolov2']
+
+    def test_models_from_model_zoo(self):
+        json_request_headers = {
+            'Content-Type': 'application/json',
+            'Accept': 'application/json'
+        }
+        pb_request_headers = {
+            'Content-Type': 'application/octet-stream',
```

Comment:
Are we only testing octet streams?

Reply:
I will have functional tests for PB payload. For model zoo tests, I'd like to keep it only use octet streams.